Makefile

```diff
@@ -23,8 +23,10 @@ binaries: \
 	_output/bin/nerdctl.lima \
 	_output/share/lima/lima-guestagent.Linux-x86_64 \
 	_output/share/lima/lima-guestagent.Linux-aarch64
+	cp -aL examples _output/share/lima
 	mkdir -p _output/share/doc/lima
-	cp -aL README.md LICENSE docs examples _output/share/doc/lima
+	cp -aL README.md LICENSE docs _output/share/doc/lima
+	ln -sf ../../lima/examples _output/share/doc/lima
```

Comment:
This breaks if you have a previous version already compiled and don't run `make clean` before building the new version:

```console
$ make all
[...]
cp -aL README.md LICENSE docs _output/share/doc/lima
ln -sf ../../lima/examples _output/share/doc/lima
ln: _output/share/doc/lima/examples: Operation not permitted
make: *** [binaries] Error 1
```

I think this is ok, but wanted to point it out, in case you want to fix it.

Reply:
Yes, I know, and not worth fixing IMHO

Reply:
Also fails to during install:

```console
$ make install
mkdir -p "/usr/local"
# Use tar rather than cp, for better symlink handling
( cd _output && tar c * | tar Cxv "/usr/local" )
x bin/
[...]
x share/doc/lima/examples: Can't replace existing directory with non-directory
[...]
tar: Error exit delayed from previous errors.
make: *** [install] Error 1
```

You have to run `make uninstall` first to fix it.

Reply:
I guess this is not our problem, but I wonder how this would affect updates via `brew`.

Reply:
Added commit `[Makefile: binaries depends on clean; install depends on uninstall](https://github.com/lima-vm/lima/pull/674/commits/d6343d5fae629212662d71c7ad25b1d18e620884)`

Homebrew is unlikely to be affected, as they have versioned directories under `Cellar`